CONTRIBUTING.rst

```diff
@@ -15,23 +15,24 @@
     specific language governing permissions and limitations
     under the License.
 
+Contributions primer
+====================
+
 .. contents:: :local:
 
-Contributions
-=============
+Contributions are welcome and are greatly appreciated! Every little bit helps, and credit will always be given.
 
-Contributions are welcome and are greatly appreciated! Every little bit helps,
-and credit will always be given.
+This document explains general contribution rules and links to detailed documentation that aim to explain
+the subject of contributions if you have not contributed to any Open Source project, but it will also help
+people who have contributed to other projects learn about the rules of that community.
```

Comment:
```suggestion
This document explains general contribution rules and links to detailed documentation that aims to explain
the subject of contributions. This is particularly useful if you are new to Open Source, but it will also help
people who have contributed to other projects learn about the rules of our community.
```

Or something similar.